src/hive/syncthing/controller.clj

```diff
@@ -0,0 +1,25 @@
+(ns hive.syncthing.controller
+  (:require [hive.syncthing.service :as syncthing.service]))
+
+
+(defn register-device
+  [device-id name]
+  (println (str "registering device id " device-id))
+  (syncthing.service/add-device {:deviceID device-id
+                                 :addresses ["dynamic"]
+                                 :introducer false
+                                 :name name})
+  true)
```

Comment:
Any reason for returning explicitly true?

Reply:
eu to meio no go-horse

Reply:
🐴